certbot/plugins/manual.py

```diff
@@ -213,17 +158,10 @@ def _perform_achall_manually(self, achall):
                 achall=achall, encoded_token=achall.chall.encode('token'),
                 port=self.config.http01_port,
                 uri=achall.chall.uri(achall.domain), validation=validation)
-        elif isinstance(achall.chall, challenges.DNS01):
+        else:  # Can be only DNS-01
             msg = self._DNS_INSTRUCTIONS.format(
```

Comment:
nit: Want to add `assert isinstance(achall.chall, challenges.DNS01)` above this line like we had for `TLSSNI01`? I think this accomplishes the same thing as your comment and will cause us to cleanly error out if something has gone horribly wrong.

Reply:
Done.